student.rb

```diff
@@ -1,8 +1,10 @@
 require './person'
 
 class Student < Person
-  def initialize(age, classroom, name, parent_permission)
-    super(age, name, parent_permission)
+  attr_accessor :classroom
+
+  def initialize(classroom, age)
+    super(age)
```

Comment:
- [ ] it would be better to add a classroom method in the `student`.also in order to achieve the `has many and belongs to` relationship also when setting the classroom for a student it also adds it to the classrooms' students

Reply:
- [ ] As the previous reviewer mentioned, Please add a classroom **method** in the `student`.also in order to achieve the `has many and belongs` to relationship also when setting the classroom for a student it also adds it to the classrooms' students.
[Hint]: It will be like that:
```ruby
def classroom=(classroom)
    @classroom = classroom
    classroom.student.push(self)
end
```